src/components/PopupBase.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { PropsWithChildren } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { css } from '../../styled-system/css'
+import { SystemStyleObject } from '../../styled-system/types'
+import { alertActionCreator as alert } from '../actions/alert'
+import { clearMulticursorsActionCreator as clearMulticursors } from '../actions/clearMulticursors'
+import { deleteResumableFile } from '../actions/importFiles'
+import { isTouch } from '../browser'
+import { AlertType } from '../constants'
+import useCombinedRefs from '../hooks/useCombinedRefs'
+import usePositionFixed from '../hooks/usePositionFixed'
+import useSwipeToDismiss from '../hooks/useSwipeToDismiss'
+import syncStatusStore from '../stores/syncStatus'
+import fastClick from '../util/fastClick'
+import CloseButton from './CloseButton'
+
+export type PopupBaseProps = PropsWithChildren<
+  {
+    /** used to cancel imports */
```

Comment:
```suggestion
    /** Used to cancel imports. */
```

Apologies for missing this in my previous suggestion, but the ESLint rules require JSDoc comments to begin with a capital letter and end with a period.